downstream/modules/platform/ref-controller-performance-improvements.adoc

```diff
@@ -0,0 +1,18 @@
+[id="ref-controller-performance-improvements"]
+
+= Performance improvements
+
+From {ControllerName} v4.3 multiple improvements have been made that support large scale deployments of {ControllerName}. 
+
+There is support for workloads with many more concurrent jobs. 
+In the past, there were issues with excessive database connections, job scheduling issues if there were thousands of pending and running jobs, and issues with successfully starting jobs when operating near 100% capacity of control nodes.
+
+Additionally, changes have been made by default to take advantage of processor capacity available on larger control nodes. 
+If you provision larger control nodes and want to run thousands of concurrent jobs you can take advantage of the followin improvements in{ControllerName}:
```

Comment:
Suggest adding a comma after "jobs" and a space before {ControllerName}:
If you provision larger control nodes and want to run thousands of concurrent jobs, you can take advantage of the followin improvements in {ControllerName}: